enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/provider/DocumentVerificationProvider.java

```diff
@@ -105,8 +112,9 @@ public interface DocumentVerificationProvider {
      * @param id Owner identification.
      * @param initAttributes Initialization attributes
      * @return Info with data related to the verification process in SDK.
-     * @throws DocumentVerificationException When an error during initializing the verification SDK occurred
+     * @throws RemoteCommunicationException In case of remote communication error
+     * @throws DocumentVerificationException In case of business logic error
      */
-    VerificationSdkInfo initVerificationSdk(OwnerId id, Map<String, String> initAttributes) throws DocumentVerificationException;
+    VerificationSdkInfo initVerificationSdk(OwnerId id, Map<String, String> initAttributes) throws RemoteCommunicationException, DocumentVerificationException;
```

Comment:
The main change of this PR is to distinguish business logic error from communication error.